content/general/concepts/abstract-factory/abstract-factory.md

```diff
@@ -0,0 +1,381 @@
+---
+Title: 'Abstract Factory Pattern'
+Description: 'The abstract factory is often thought of a factory of factory patterns. Much like the factory pattern, the abstract factory relies on a common interface and defers instantiation logic to its concrete model classes.'
```

Comment:
For entry metadata, we ideally want the `Description` to have the following things: 

- Is 150 characters or less.
- Begins with a verb in the present-tense. (i.e. "Runs", "Describes", "Works as", etc.)
- (optional) Closely resembles the first 1-2 sentences of the entry's opening paragraph.

Meeting these two (or three) criteria improves the entry page's SEO.